.github/workflows/shared-subsystem-auto-deploy.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
   terraform:
     uses: web3-storage/ipfs-elastic-provider-infrastructure/.github/workflows/terraform-aws-shared-workflow.yaml@main 
     with:
-      options: -var profile=ipfs
+      options: -var profile=ipfs -var grafana_auth=${{ secrets.GRAFANA_APP_KEY }}
```

Comment:
will this be masked out properly when passed to the shared-workflow?

Reply:
Isn't this required by the `observability-grafana` root instead of the `shared-subsystem` root?

Reply:
GitHub actions doesn't print secrets when running script, it shows like: *******

Reply:
and oooops.. correcting the workflow